csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.cs

```diff
@@ -92,6 +92,18 @@ private static SessionOptions MakeSessionOptionWithCpuProvider()
             return options;
         }
 
+        /// <summary>
+        /// A helper method to construct a SessionOptions object for Nuphar execution
+        /// </summary>
+        /// <returns>A SessionsOptions() object configured for execution with Nuphar</returns>
+        public static SessionOptions MakeSessionOptionWithNupharProvider()
```

Comment:
>MakeSessionOptionWithNupharProvider [](start = 37, length = 35)

nit: MakeSessionOptionsWithNupharProvider

Reply:
Other providers in the file seems not using plural form. Will update if needed once synced to latest master.

---
In reply to: [312414773](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312414773) [](ancestors = 312414773)